packages/connect/src/auth.ts

```diff
@@ -14,6 +14,7 @@ export interface AuthOptions {
   redirectTo?: string;
   manifestPath?: string;
   finished?: (payload: FinishedData) => void;
+  cancelled?: () => void;
```

Comment:
I would really press for naming syntax to be `onCancelled`. Thoughts @hstove @kyranjamie ?

Reply:
Changed :+1:

Reply:
Wouldn't it be the present tense at this point. `onCancel` ?

Reply:
@kyranjamie yes that makes more sense

Reply:
@kyranjamie @hstove could we add `onFinish`?